ocs_ci/templates/workloads/fio/benchmark_fio_cmp.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: fio-benchmark
+  namespace: my-ripsaw
+spec:
+  elasticsearch:
+    server: 10.0.78.167
+    port: 9200
+    parallel: true
+  clustername: env_ocpversion_ocsversion
+  test_user: interface_iopattern
+  workload:
+    name: "fio_distributed"
+    args:
+      prefill: true
+      samples: 1
+      servers: 5
+      pin_server: ''
+      jobs:
+        - randwrite
+        #- randread
+      bs:
+        - 1024KiB
+        #- 64KiB
```

Comment:
can we remove these instead of commenting.

Reply:
fixed.